libs/sdk/e2e-schematics/src/utils/find-module.ts

```diff
@@ -36,15 +36,21 @@ export function findDeclaringModule(
   componentPath: string,
 ): { filepath: string; module: DecoratedClass } | null {
   let result: { filepath: string; module: DecoratedClass } | null = null;
+
   const componentSource = readSourceFile(tree, componentPath);
   const componentClass = findComponentClass(componentSource);
+
   if (componentClass) {
-    if (componentClass.properties['standalone']) {
+    const standalone = componentClass.properties['standalone'];
+
+    // If "standalone" is omitted, the component is standalone.
+    if (standalone === undefined) {
```

Comment:
In theory - someone could still put `standalone: true`, correct? I know we are removing them and shouldn't do that going forward - but is there anything stopping someone from doing it? This current statement would only ever treat something as standalone if the property isn't there.

Reply:
It's a good call out, but I believe these scripts are only executed when we want to create a new e2e project, so I didn't think we needed to worry about that edge case?

Reply:
Fair.